README.md

```diff
@@ -61,7 +60,7 @@ Niquests officially supports Python or PyPy 3.7+.
 
 ## Supported Features & Best–Practices
 
-Niquests is ready for the demands of building robust and reliable HTTP–speaking applications, for the needs of today.
+Niquests is ready for the demands of building modern, robust and reliable HTTP–speaking applications.
```

Comment:
"modern" only tend to "trigger" people. Maybe choosing something else..?

Reply:
Given the ability to use concurrency, perhaps eliminate modern entirely and use "scalable".